### PR TITLE
Too mortal

### PR DESCRIPTION
**Pull request by jonsterling**

### CODE REVIEW

File: src/core/Val.ml [hunk 1]
```diff
@@ -1907,7 +1907,7 @@ struct
           let body =
             lazy begin
               let func = Value.act phi func in
-              apply func el
+              apply func @@ Value.act phi el
```
Comment (jonsterling) [inline src/core/Val.ml]: @favonia This seemed like a bug, but not sure. what do you think?

Comment (favonia) [reply src/core/Val.ml]: Indeed! However, is there a reason not to use the helper functions in Face.ml?

Comment (jonsterling) [reply src/core/Val.ml]: The main reason is I didn't understand how to use those functions.

Comment (jonsterling) [reply src/core/Val.ml]: Let me update it

File: src/core/Quote.ml [hunk 2]
```diff
@@ -397,7 +397,7 @@ struct
       let tr, tr' = equate_dir env info0.dir info1.dir in
       let _ = equate_ty env info0.univ info1.univ in
       let ty = equate_neu env info0.ty info1.ty in
-      let ty_val = make @@ Up {ty = info0.univ; neu = info0.ty; sys = []} in
+      let ty_val = reflect info0.univ info0.ty @@ ValSys.from_rigid info0.ty_sys in
```
Comment (jonsterling) [inline src/core/Quote.ml]: Taste it. This was the problem.

**Pull request merged**